packages/server/modules/workspaces/events/eventListener.ts

```diff
@@ -41,13 +41,15 @@ export const onProjectCreatedFactory =
     const workspaceMembers = await getWorkspaceRoles({ workspaceId })
 
     await Promise.all(
-      workspaceMembers.map(({ userId, role: workspaceRole }) =>
-        grantStreamPermissions({
+      workspaceMembers.map(({ userId, role: workspaceRole }) => {
+        if (workspaceRole === Roles.Workspace.Guest) return false
```

Comment:
cc @cdriesler

Reply:
@cdriesler There's no difference, the point is to return out. The return value is irrelevant, it'll just get awaited and then forgotten about.

I'd rather not add extra looping throughout the entire array just to filter out things. If you think the `return false` is confusing, I can make it a void return